docs/static/releasenotes.asciidoc

```diff
@@ -37,6 +38,86 @@ This section summarizes the changes in the following releases:
 * <<logstash-8-0-0-alpha1,Logstash 8.0.0-alpha1>>
 
 
+[[logstash-8-8-1]]
+=== Logstash 8.8.1 Release Notes
+
+* Remove obsolete notice when using plugins with version < 1.0.0 https://github.com/elastic/logstash/pull/15077[#15077]
+* Docs: Add instructions to verify Docker install images (#15058) (#15064) https://github.com/elastic/logstash/pull/15064[#15064]
+
```

Comment:
This fix is transitive since Monitoring and Central Management connect through the ES Client from an ES Output plugin, but should be called out in the release notes.

```suggestion
 * Fixes a regression introduced in 8.8.0 which could prevent Monitoring or Central Management from establishing a connection to Elasticsearch in some SSL configurations https://github.com/elastic/logstash/issues/15068[#15068]

```